### PR TITLE
Add missing RBAC permissions

### PR DESCRIPTION
**Pull request by myoung34**
Resolves https://github.com/containous/traefik-helm-chart/issues/254

### CODE REVIEW

**Comment by SantoDE** [comment]
We should merge that after we officialy bumped the chart with 2.3. Don't you think so?

**Comment by myoung34** [comment]
AFAIK this is a problem in 2.2 as well

**Comment by SantoDE** [comment]
But it can't for the `ingressClass`, as 2.2 is not loading it. Maybe for Ingress with a changed apiGroup tough. Maybe, we could limit the PR to this then? Because, RBAC should be in #254 then I guess. Right?

**Comment by myoung34** [comment]
I'm not sure to be honest, I'm still on 2.3, but I can try to find out for sure this weekend and scope this to #246

**Comment by SantoDE** [comment]
Hey @myoung34,

thanks! Any updates?

File: traefik/templates/rbac/clusterrole.yaml [hunk 1]
```diff
@@ -47,4 +47,14 @@ rules:
...
+  - apiGroups:
+      - extensions
+      - networking.k8s.io
+    resources:
+      - ingresses
+      - ingressclasses
+    verbs:
+      - get
+      - list
+      - watch
```
Comment (DirtyCajunRice) [inline traefik/templates/rbac/clusterrole.yaml:59]: This creates duplication. The addition should just be to the existing apigroups

**Review by DirtyCajunRice — CHANGES_REQUESTED** [review]
This should be removed in favor of augmenting the existing apigroups

https://github.com/traefik/traefik-helm-chart/blob/92517a02f5f07ff89cb1129dd93cb3387cd1d031/traefik/templates/rbac/clusterrole.yaml#L22-L29

would have the `networking.k8s.io` apiGroup added and the `ingressclasses` resource added.

as for 2.3 conversation: If you are running the RC of traefik you need to manually add the apiGroups. The chart should match the release of 2.3 in permissions and would happen at the same time as #246

File: traefik/templates/rbac/clusterrole.yaml (hunk 1 above)
Comment (notsureifkevin) [reply traefik/templates/rbac/clusterrole.yaml:59]: `extensions` -> `ingresses` -> `get`, `watch`, `list` appear to be covered already, however, `networking.k8s.io` is not, and could be added to the existing definition at `apiGroups[1]`

**Comment by SantoDE** [comment]
As #267 is merged, I'll close that one.

Thanks 👍

**Pull request closed without merging**